src/components/markdown_editor/__snapshots__/markdown_editor.test.tsx.snap

```diff
@@ -191,6 +191,7 @@ exports[`OuiMarkdownEditor is rendered 1`] = `
   >
     <div
       class="ouiMarkdownEditorDropZone"
+      role="presentation"
```

Comment:
@andreymyssak Can you explain the significance of adding these `role`s for this update (other than being nice semantically and for a11y)?

Reply:
@joshuarrrr Since this is a snapshot, it has been updated according to the latest changes in `react-dropzone`. Here is the PR https://github.com/react-dropzone/react-dropzone/issues/1152 where you can find the reason for this change

Reply:
perfect, thanks!